src/smartcard/util/__init__.py

```diff
@@ -22,13 +22,15 @@
 Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 """
 
+from __future__ import annotations
+
 PACK = 1
 HEX = 2
 UPPERCASE = 4
 COMMA = 8
 
 
-def padd(bytelist, length, padding='FF'):
+def padd(bytelist: list[int], length: int, padding: str = 'FF'):
```

Comment:
PEP8 https://peps.python.org/pep-0008/#other-recommendations: Don’t use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

Use `str='FF'` without spaces.

Reply:
That is true for an unannotated function.

From that same document:

> When combining an argument annotation with a default value, however, do use spaces around the `=` sign:
>
> ```python
> # Correct:
> def munge(sep: AnyStr = None): ...
> def munge(input: AnyStr, sep: AnyStr = None, limit=1000): ...
>
> # Wrong:
> def munge(input: AnyStr=None): ...
> def munge(input: AnyStr, limit = 1000): ...
> ```